src/main/java/com/kruize/recommendations/application/ApplicationRecommendationsImpl.java

```diff
@@ -179,15 +179,21 @@ public double getCpuRequests(String applicationName) throws NoSuchApplicationExc
         }
     }
 
-    /* If all the instances of the application are idle, return idle */
+    /* If any instances of the application is running, return running */
     public String getStatus(String applicationName)
     {
+        boolean applicationIdle = false;
         for (MetricsImpl metric: applicationMap.get(applicationName))
         {
             if (metric.getStatus().equals("running"))
                 return "running";
+            else if (metric.getStatus().equals("idle"))
```

Comment:
No need for `else` here as we return immediately.
```
if (metric.getStatus().equals("running"))
      return "running";
if (metric.getStatus().equals("idle"))
      applicationIdle = true;

Reply:
Oops, missed this. Fixed in the newest commit.